sktime/forecasting/sarimax.py

```diff
@@ -221,3 +221,47 @@ def _extract_conf_int(prediction_results, alpha) -> pd.DataFrame:
         conf_int.columns = ["lower", "upper"]
 
         return conf_int
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+            There are currently no reserved values for forecasters.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+        return [
+            {
```

Comment:
To reduce the load on the testing system, It would be better to use only the last two more complex parameter sets, since the first two are almost covered by the others. @fkiraly what is your opinion?

Reply:
agreed (maybe ensure that list/tuple is also tested both)